packages/components/src/TCRUserDashboard/DashboardStyledComponents.tsx

```diff
@@ -545,7 +544,7 @@ export const StyledDropdownArrow = styled.div`
 export const StyledFromBalance = styled.div`
   border: 1px solid ${colors.accent.CIVIL_GRAY_3};
   border-radius: 3px;
-  margin-top 5px;
+  margin-top: 5px;
```

Comment:
Good catch god dang how many of these are lurking.

Out of curiosity are you finding these manually or do you have some IDE linting that's picking it up or something?